packages/utils/src/helpers.spec.ts

```
@@ -1,4 +1,57 @@
-import { splitArrayToChunks } from './helpers.js'
+import { awaitForAll , splitArrayToChunks} from './helpers.js'
```

ESLint

[error] Replace `·,·splitArrayToChunks` with `,·splitArrayToChunks·`

(prettier/prettier)